src/components/molecules/GeoJsonForm.vue

```diff
@@ -1,8 +1,30 @@
 <script lang="ts">
-import { defineComponent } from '@vue/composition-api';
+import { defineComponent, ref, watch } from '@vue/composition-api';
+import { useMap, geoShape } from '@/store';
 
 export default defineComponent({
   name: 'GeoJsonForm',
+
+  setup() {
+    // eslint-disable-next-line no-var
+    var geoJsonShape = ref();
```

Comment:
```suggestion
    const geoJsonShape = ref();
```
Why use `var` here?

Reply:
Sincerely no idea why I had that as var. It has been changed.